Migrate Cloudcraft documentation from Help Scout to Datadog (#21358)
DataDog/documentation: content/en/cloudcraft/faq/why-cant-export-diagram-to-terraform.md

---
title: Why can't I export diagrams as Terraform code anymore?

> kind: guide

Reviewer: ```suggestion
kind: faq
```

---

> As the **Export to Terraform** feature relied on a third-party provider, which limited our ability to support and improve its capabilities, Cloudcraft decided to remove the ability to export diagrams to Terraform code.
> 
> We plan to bring this feature in-house at a later date, but there is no timeline commitment at this point.

Reviewer: ```suggestion
The **Export to Terraform** feature relied on a third-party provider, which limited our ability to support and improve its capabilities. As a result, Cloudcraft decided to remove the ability to export diagrams to Terraform code.
```
